helm/pinot/templates/broker/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,29 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if .Values.broker.disruptionBudget.enabled -}}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "pinot.broker.fullname" . }}
+spec:
+  minAvailable: {{ .Values.broker.disruptionBudget.minAvailable }}
```

Comment:
It would be nice to also have the ability to define a `maxUnavailable`.

This is specifically useful in servers, where if we know the min replica size for all tables, we can configure pdb as `maxUnavailable: replicaSize - 1`

Reply:
added

Reply:
I've left the defaults values for broker and controller at `50%`, although `replicaSize - 1` would make sense too

The number for servers would be `maxUnavailable: $tableReplicas - 1` or similar, which isn't easily available here so I just set it to 1 in the default values